packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -158,20 +119,18 @@
   justify-content: space-between;
   box-sizing: border-box;
   font-size: $iui-font-size-subheading;
+}
 
-  @at-root .iui-dialog-draggable & {
-    user-select: none;
-    font-size: $iui-font-size-leading;
-    padding: round($iui-baseline * 0.5) $iui-m;
-    cursor: grab;
-    @include themed {
-      background-color: t(iui-color-background-3);
-      border-bottom: 1px solid t(iui-color-background-border);
-    }
+@mixin iui-dialog-title-bar-draggable {
```

Comment:
Why changed?

Reply:
To reduce nesting and decouple it from the modifier class. The styling is not directly related to draggability (other than the cursor) so it could even be useful for custom dialogs (in itwinjs for example). Just not sure about the name.

Reply:
Maybe `iui-filled`, `iui-bordered`? Idk, @FlyersPh9 always has ideas 😀

Reply:
At the moment I think this is only intended for a draggable dialog because it gives the user a clear area where they can click & drag.  I also think only a draggable dialog can be resized.  I'm ok with the current name & `@mixin iui-dialog-title-bar-filled`, but if we want to think of alternatives maybe `@mixin iui-dialog-title-bar-controllable` or something along those lines?

Reply:
Changed to `iui-dialog-title-bar-filled`